internal/utils/net.go

```diff
@@ -0,0 +1,28 @@
+package net
+
+import (
+	"net"
+	"net/netip"
+
+	"github.com/free5gc/udm/internal/logger"
+)
+
+func BindingLookup(bindIP string) string {
```

Comment:
You can probably only have a single function here, and have func (c *Config) GetSbiBindingIP() returns netip.Addr instead of a string, and instead of func (c *Config) GetSbiBindingAddr doing the netip.ParseAddr.
Maybe you should add some Info-level log which gives info on which IP was selected from the Lookup?
Are you also guaranteed that len(ips) > 0s when err != nil?

Reply:
`len(ips) > 0` if `net.LookupIP` does not return a error

Reply:
I agree, I refactor it into one function, so I directly move this function into _internal/context/context.go_ module
I add a Info-level log if the IP was selected from the Lookup